Meadowlark-js/backends/meadowlark-mongodb-backend/package.json

```diff
@@ -19,6 +19,7 @@
     "build:copy-non-ts": "copyfiles -u 1 -e \"**/*.ts\" \"src/**/*\" dist --verbose"
   },
   "dependencies": {
+    "@edfi/meadowlark-authz-server": "0.2.0-dev.0",
```

Comment:
I'm not a fan of abbreviations in code - don't like anything that adds to cognitive load - but figured it was okay to use standard "authorization" abbreviation in package name. Or not.

Reply:
I've never seen authz before, The abbreviation for has always been 'auth' to me. A quick google showed there are different abbreviations for authorization and authentication. So that's one of those 'you learn something new every day'

I'm with you on abbreviations, but I think this one is fine